app/mailers/notification_mailer.rb

```diff
@@ -1,10 +1,23 @@
 class NotificationMailer < ActionMailer::Base
   default from: 'no-reply@gitreports.com'
 
-  def issue_submitted_email(repo_id, issue_id)
+  def issue_submitted_email(repo_id, issue_id, submitter_info = {})
+    defaults = {
```

Comment:
Also fwiw this isn't doing what you probably think anyway 🙂  you would want:

```ruby
defaults = {
  submitter_name: nil,
  submitter_email: nil
}
```

As it is, you're using dynamic hash keys, so ruby will find the value of the instance variables `@submitter_name` and `@submitter_email` (which are both obviously null at this point) and use the values as the hash keys. So what's written here is actually equivalent to:

```ruby
defaults = {
  nil => nil,
  nil => nil
}
```